app/controllers/policies_controller.rb

```diff
@@ -93,6 +96,8 @@ def updated_can_publish_immediately(resource, projects)
       #FIXME: need to use User.current_user here because of the way the function tests in PolicyControllerTest work, without correctly creating the session and @request etc
     elsif projects.any? { |p| p.asset_gatekeepers.any? } && !projects.any? { |p| User.current_user.person.is_asset_gatekeeper?(p) }
       false
+    elsif !projects.any?
+      !resource.gatekeeper_required?
```

Comment:
I don't see a double negative there... do you mean using `projects.empty?` instead of `!projects.any?`?
I used `!projects.any?` because it is used before my changes in that same context, so using the same thing would make it easier to track... but happy to change it if that's what you meant?

Reply:
yes, it would be clearer using `projects.empty?`

Reply:
or you could try and refactor the logic to use unless instead. try and avoid `if !x` https://rubystyle.guide/#unless-for-negatives

Reply:
I think it is a bit simpler now, see [d5da530](https://github.com/seek4science/seek/pull/1484/commits/d5da5309b8a1a7bda2fb535a7e856fe1ff319854).